src/components/Page.js

```diff
@@ -13,11 +25,18 @@ class Page extends React.Component {
   };
 
   render() {
-    const { name, children } = this.props;
-    const _name =
-      name === 'Symbols' ? 'Symbols (renamed to avoid conflict)' : name;
+    const { name, children, style, ...otherProps } = this.props;
 
-    return <page name={_name}>{children}</page>;
+    const pageProps = {
+      name: name === 'Symbols' ? 'Symbols (renamed to avoid conflict)' : name,
+      ...otherProps,
```

Comment:
can we avoid using `...` twice? just pass the `name` prop without merging it with the `otherProps`. Same for `style` I'd say

Reply:
I can do, but was trying to avoid changing the snapshots for the existing tests.

The following removes the additional spread operator, but causes the existing tests to fail because ```style``` becomes an property with undefined value.  I don't know how the underlying Sketch ```page``` component handles that and it could have an unintended side effect.

Making ```style``` conditionally included in the ```pageProps``` var avoids this possibility.

Let me know what you think, I'll update the PR if you want.

```
  render() {
    const { name, children, style, ...otherProps } = this.props;
    const _name = name === 'Symbols' ? 'Symbols (renamed to avoid conflict)' : name;
    const _style = StyleSheet.flatten(style);

    return <page name={_name} style={_style} {...otherProps}>{children}</page>;
  }
```

Reply:
yeah I think it's fine, just update the existing snapshots.

Have you tested manually that it worked? (I usually use the `basic-setup` example and use `../../lib/` instead of `react-sketchapp` so I can test my changes directly)

Reply:
Yep, tested on Basic setup, although that doesn't specifically allow me to test the changes.  I'm happy the unit test validates the intention.